src/makeRobot.js

```diff
@@ -39,6 +39,66 @@
  */
 function makeRobot(name, wheels, version) {
   // write code here
+  const robot = {
+    'name': name,
+    'wheels': wheels,
+    'version': version,
+    coords: {
+      x: 0,
+      y: 0,
+    },
+
+    get info() {
+      // по другому линтер не пропускает
+      const resultStr
+  = `name: ${this.name}, chip version: ${this.version}, wheels: ${this.wheels}`;
```

Comment:
Use string concatenation:
```suggestion
      return `name: ${this.name}, `
        + `chip version: ${this.version}, `
        + `wheels: ${this.wheels}`;
```